### PR TITLE
Add 2021 imagery for King County

### PR DESCRIPTION
**Pull request by aaronlidman**
Screen Shot 2022-05-20 at 3 35 34 PM

Screen Shot 2022-05-20 at 3 38 13 PM

### CODE REVIEW

**Comment by cicku** [comment]
Just a quick question since I’m not near any computer, is there any boundary change comparing with 2019?

File: sources/north-america/us/wa/King_WA_2021.geojson [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+        "icon": "https://kingcounty.gov/img/KClogo-horiz-black-lg.gif",
+        "url": "https://gismaps.kingcounty.gov/arcgis/rest/services/BaseMaps/KingCo_Aerial_2021/MapServer/WMTS/tile/1.0.0/BaseMaps_KingCo_Aerial_2021/default/default028mm/{zoom}/{y}/{x}",
+        "max_zoom": 20,
+        "license_url": "https://kingcounty.gov/services/gis/Maps/terms-of-use.aspx",
+        "country_code": "US",
+        "type": "tms",
+        "start_date": "2021",
+        "end_date": "2021",
+        "description": "2021 orthoimagery for King County in the State of Washington",
+        "category": "historicphoto",
```
Comment (cicku) [inline sources/north-america/us/wa/King_WA_2021.geojson]: This is the current photo, you should not set it as historic

Comment (aaronlidman) [reply sources/north-america/us/wa/King_WA_2021.geojson]: 👍🏻 updated to `photo`

**Comment by cicku** [comment]
Can you reuse the boundary in [2019 file](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/north-america/us/wa/King_WA_2019.geojson)?

**Comment by aaronlidman** [comment]
> Just a quick question since I’m not near any computer, is there any boundary change comparing with 2019?

Yes very slightly, in the north and south. I modified the 2019 boundary just in the new coverage areas. Here's a gif showing the difference.

compare_king_co

> Can you reuse the boundary in [2019 file](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/north-america/us/wa/King_WA_2019.geojson)?

Yes I can do that but it would clip out the new coverage, do we want that?

**Comment by cicku** [comment]
Yes please do it, that small area 
image
 can be covered when the zoom level is not at 20.

Once you finish this I will merge it.

**Comment by cicku** [comment]
Thank you for the help!

**Pull request merged**